config/compilerEnv.cmake

```diff
@@ -103,14 +103,21 @@ macro(dbsSetupCompilers)
   # In component_macros.cmake, this target property will be set:
   # INTERPROCEDURAL_OPTIMIZATION_RELEASE;${USE_IPO}
 
-  set(USE_IPO ON)
-  if( CRAY_PE )
-    set( USE_IPO OFF )
-  elseif( "${SITENAME}" STREQUAL "Moonlight" )
-    set( USE_IPO OFF )
-  endif()
-  set( USE_IPO ${USE_IPO} CACHE BOOL "Use IPO?" FORCE )
-
+  #if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.9.0)  
```

Comment:
It looks like cmake < 3.9 could still be supported if this if statement was uncommented.
Will this comment merely be removed at some point?